app/views/group_offer_categories/_form.html.slim

```diff
@@ -0,0 +1,13 @@
+= simple_form_for(@group_offer_category) do |f|
+  = simple_error_notice f
+  - if policy(GroupOffer).supervisor_privileges?
```

Comment:
Since categories are restricted to superadmins, this should not be needed. Also, the restrictions should be based on category policies and not group offer ones.